api/pom.xml

```diff
@@ -110,6 +110,33 @@
 				<filtering>true</filtering>
 			</testResource>
 		</testResources>
+
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>
```

Comment:
I thought this was already happening! I think there's more to define if that's the case, see one at https://github.com/openmrs/openmrs-core/blob/master/pom.xml